models/room.py

```diff
@@ -0,0 +1,86 @@
+"""Room, RoomType, Facility, Feature, FeatureToRoomType models."""
+import enum
+
+from sqlalchemy import (CheckConstraint, Column, Enum, Float, ForeignKey,
+                        Integer, String)
+
+from db import Base
+
+
+class NumberOfBeds(str, enum.Enum):
+    """Possible options for NumberOfBeds."""
+
+    single_bed = "single_bed"
+    double_beds = "double_beds"
+    twin_beds = "twin_beds"
+    king_size_bed = "king_size_bed"
+
+
+class RoomBookingStatus(str, enum.Enum):
+    """Possible options for RoomBookingStatus."""
+
+    vacant = "vacant"
+    booked = "booked"
+    occupied = "occupied"
+
+
+class RoomCleanlinessStatus(str, enum.Enum):
+    """Possible options for RoomCleanlinessStatus."""
+
+    clean = "clean"
+    dirty = "dirty"
```

Comment:
Added enums for possible booking statuses, cleanliness statuses and (as it was previously) number of beds in the room